client/components/HomePage/LoginPage.js

```diff
@@ -49,9 +49,16 @@ class LoginPage extends Component {
   render() {
     return (
       <div className="auth-page">
-        <img src={andela_logo} alt="Andela logo" />
-        <img src={signin_btn} alt="Click to sign in" onClick={this.onSignIn} />
-        <h1>Work Hard! Code Hard! Play Harder!!!</h1>
+        <img className="andela" src="http://res.cloudinary.com/proton/image/upload/v1510947117/Condensed-white_qudkxm.png" />
+        <div className="welcome-text">
+          <p> GET CLOSER TO YOUR <span className="bold">SOCIAL MEETUP</span> </p>
+        </div>
+        <div className="image-holder">
+          <img src="http://res.cloudinary.com/proton/image/upload/v1509311455/andrew-ly-152166_w5ffs9.jpg" className="featured-meetup" />
+        </div>
+        <div className="button">
+          <button type="submit" id="button">SIGN IN</button>
```

Comment:
There should be an `onClick={this.onSignIn}` to sign in the user

Reply:
I suppose this should be done and tested as well

Reply:
Resolved. We haven't started writing tests yet.